docker/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-ARG BASE=zilliqa/scilla:v0.12.0rc1
+ARG BASE=zilliqa/scilla:a8d0c412
```

Comment:
Should this line be merged? The commit that this refers to is now merged to `master`.

Reply:
I will revert this commit before the merge with the new scilla tag. Only kept it for testing.